.travis.yml

```diff
@@ -11,12 +11,17 @@ matrix:
     - node_js: 8 # Version used to deploy to npm registry
       env: BUILD_ENV=production
 
-cache:
-  directories:
-    - ~/.npm
+cache: yarn
+
+before_install:
+  - yarn global add greenkeeper-lockfile@1
 
 before_script:
-  - NODE_ENV=$BUILD_ENV npm run build
+  - greenkeeper-lockfile-update
+  - NODE_ENV=$BUILD_ENV yarn build
```

Comment:
It's a bit silly, but according to https://travis-ci.org/thelounge/lounge/jobs/334724018#L531, `npm test` is still being run. We probably want to explicitly define `yarn test` in `script`, and/or let them know that `npm` is incorrectly used.

Reply:
Same with https://travis-ci.org/thelounge/lounge/jobs/334724018#L449-L450, they should print `yarn --version` instead.
Okay, these 2 might be worth opening a support ticket or something.

As part of this, it's also worth checking with them if releasing will correctly be done by `yarn` (but still on npm registry).

Reply:
I created https://github.com/travis-ci/travis-build/pull/1299 for that. If that's not accepted, I will change our script.

Reply:
Update: they mentioned they'll merge this and deploy tomorrow. 👌